.github/workflows/build.yml

```diff
@@ -54,6 +54,19 @@ jobs:
               - 'libs/content-watcher-lib/**'
               - 'libs/storage/**'
 
+  test-start-sh:
+    name: 'Bats start.sh Tests'
+    runs-on: ubuntu-latest
+    needs: service-matrix
+    strategy:
+      fail-fast: false
+    steps:
+      - name: 'Bats start.sh Tests'
+        if: ${{ needs.service-matrix.outputs.changes.common }}
```

Comment:
Pretty sure you need `if: ${{ needs.service-matrix.outputs.changes.common == true }}`, this was the problem the last time I debugged this...